odata-parser.js

```diff
@@ -1197,21 +1152,44 @@
                   case ")":
                     return ")";
 
+                  case "*":
+                    return "*";
+
                   case "+":
                     return "+";
 
                   case ",":
                     return ",";
 
+                  case "/":
+                    return "/";
+
+                  case ":":
+                    return ":";
+
                   case ";":
                     return ";";
 
                   case "=":
                     return "=";
 
+                  case "?":
+                    return "?";
+
+                  case "@":
+                    return "@";
+
+                  case "[":
+                    return "[";
+
+                  case "]":
+                    return "]";
+
```

Comment:
Can't we use fallthrough here?

Reply:
How would fallthrough help here? It should be possible to add it to the code generation but I don't see how it would be used

Reply:
Just rather than repeating all of the chars, add all of the cases and have it echo the char itself. Super minor though.